assets/stylesheets/base/_utilities.scss

```diff
@@ -398,97 +156,67 @@ a.underline,
   border-bottom: 1px solid $gray-lighter;
 }
 
-.pull-right.btn {
-  margin-left: 5px;
-}
+/* Corners */
 
-// Square corners
 .square-corners {
   border-radius: 0;
 }
 
-//
-// Bootstrap size based pull-left and pull-right
-//
-
-@media (min-width: $screen-lg-min) {
-  .lg-pull-right {
-    float: right;
-    &.btn {
-        margin-left: 5px;
-    }
-  }
-  .lg-pull-left {
-    float: left;
-  }
+.round-corners {
+  border-radius: $border-radius-base;
 }
 
-@media (min-width: $screen-md-min) {
-  .md-pull-right {
-    float: right;
-    &.btn {
-        margin-left: 5px;
-    }
-  }
-  .md-pull-left {
-    float: left;
-  }
-}
+/* Stackable floats */
+
 
-@media (min-width: $screen-sm-min) {
-  .sm-pull-right {
-    float: right;
-    &.btn {
-        margin-left: 5px;
+@mixin floats {
+  $sizes: (
+    $screen-sm-min: "sm",
+    $screen-md-min: "md",
+    $screen-lg-min: "lg"
+  );
+
+  $directions: ("right", "left");
+
+  @each $screen, $size in $sizes {
+    @each $direction in $directions {
+      @media only screen and (min-width: $screen) {
+        .#{$size}-pull-#{$direction} {
+          float: #{$direction};
+        }
+      }
     }
   }
-  .sm-pull-left {
-    float: left;
-  }
 }
 
+@include floats;
```

Comment:
You've got a couple mixins here that are only used in one place. Does that make sense? Should mixins belong here and not in the mixins module?